pkg/porter/helpers.go

```diff
@@ -90,3 +93,22 @@ func (p *TestMixinProvider) GetVersion(m mixin.Metadata) (string, error) {
 func (p *TestMixinProvider) Install(o mixin.InstallOptions) (mixin.Metadata, error) {
 	return mixin.Metadata{Name: "exec", Dir: "~/.porter/mixins/exec"}, nil
 }
+
+// If you seek a mock cache for testing, use this
+type mockCache struct {
+	findBundleMock        func(string) (string, bool, error)
```

Comment:
You don't have to change this now. I just wanted to share that my plan for our test helpers was that they are usable for downstream consumers. So that they can write unit tests too.

That's why you can do things like `porter.NewTestPorter` from our integration tests in /tests/rebuild_test.go even though it's outside of the pkg/porter package. Same for TestContext and TestConfig, etc.

So eventually we may need something like TestCache? or exposing this mockCache struct to help them out too.

Reply:
Yeah, that sounds like a good idea. I'll do a follow up for that if that's cool.